go-controller/pkg/cni/cni.go

```diff
@@ -101,7 +101,8 @@ func (pr *PodRequest) checkOrUpdatePodUID(pod *kapi.Pod) error {
 		pr.PodUID = string(pod.UID)
 	} else if string(pod.UID) != pr.PodUID {
 		// Exit early if the pod was deleted and recreated already
-		return fmt.Errorf("pod deleted before sandbox %v operation began", pr.Command)
+		return fmt.Errorf("pod deleted before sandbox %v operation began. Request Pod UID %s is different from the"+
+			" informer/API servers (%s)", pr.Command, pr.PodUID, pod.UID)
```

Comment:
```suggestion
			" what informer/API holds (%s)", pr.Command, pr.PodUID, pod.UID)
```
nit ^
I think its a tiny bit confusing when you say different from the informer/API servers... feels like their version of the pod UID bit is missing